### PR TITLE
Performance: optimize IDs read of ElasticSearch storage options(6 and 7)

### PR DESCRIPTION
**Pull request by wu-sheng**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

Performance: optimize IDs read of ElasticSearch storage options(6 and 7). Use the physical index rather than the template alias name.

### CODE REVIEW

File: oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/HistoryDeleteEsDAO.java [hunk 1]
```diff
@@ -73,4 +74,39 @@ public void deleteHistory(Model model, String timeBucketColumnName, int ttl) thr
...
+            try {
+                indexes = client.retrievalIndexByAliases(tableName);
+            } catch (IOException e) {
+                log.error(e.getMessage(), e);
+                return;
+            }
+
+            for (String index : indexes) {
+                indices.add(index);
+            }
```
Comment (kezhenxu94) [inline oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/HistoryDeleteEsDAO.java:108]:
```suggestion
            indices.addAll(indexes);
```

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Review by kezhenxu94 — APPROVED** [review]

File: oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/MetricsEsDAO.java [hunk 2]
```diff
@@ -42,16 +49,54 @@ protected MetricsEsDAO(ElasticSearchClient client,
...
+
+        // The groupIndices mostly include one or two group,
+        // the current day and the T-1 day(if at the edge between days)
+        List<Metrics> result = new ArrayList<>(metrics.size());
+        groupIndices.forEach((tableName, metricList) -> {
+            String[] ids = metrics.stream()
+                                  .map(item -> IndexController.INSTANCE.generateDocId(model, item.id()))
+                                  .toArray(String[]::new);
+            try {
+                SearchResponse response = getClient().ids(tableName, ids);
```
Comment (Ax1an) [inline oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/MetricsEsDAO.java:90]:
```suggestion
        groupIndices.forEach((tableName, metricList) -> {
            String[] ids = metricList.stream()
                                  .map(item -> IndexController.INSTANCE.generateDocId(model, item.id()))
                                  .toArray(String[]::new);
            try {
                SearchResponse response = getClient().ids(tableName, ids);
```

Comment (wu-sheng) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/MetricsEsDAO.java:90]: Good cache.

Comment (wu-sheng) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/MetricsEsDAO.java:90]: Open this, https://github.com/apache/skywalking/pull/7309

**Pull request merged**